src/main/java/org/datadog/jenkins/plugins/datadog/DatadogPluginManagement.java

```diff
@@ -0,0 +1,144 @@
+package org.datadog.jenkins.plugins.datadog;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import hudson.ExtensionList;
+import hudson.model.ManagementLink;
+import hudson.security.Permission;
+import jenkins.model.Jenkins;
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.datadog.jenkins.plugins.datadog.flare.FlareContributor;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.interceptor.RequirePOST;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletResponse;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+@Extension
+public class DatadogPluginManagement extends ManagementLink {
+
+    private static final Logger LOGGER = Logger.getLogger(DatadogPluginManagement.class.getName());
+
+    private final List<FlareContributor> contributors;
+
+    public DatadogPluginManagement() {
+        contributors = new ArrayList<>(ExtensionList.lookup(FlareContributor.class));
+        contributors.sort(Comparator.comparingInt(FlareContributor::order));
+    }
+
+    public List<FlareContributor> getContributors() {
+        return contributors;
+    }
+
+    @CheckForNull
+    @Override
+    public String getIconFileName() {
+        return "/plugin/datadog/icons/dd_icon_rgb.svg";
+    }
+
+    @CheckForNull
+    @Override
+    public String getDisplayName() {
+        return "Datadog";
+    }
+
+    @CheckForNull
+    @Override
+    public String getUrlName() {
+        return "datadog";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Datadog Plugin Troubleshooting";
+    }
+
+    @NonNull
+    @Override
+    public Category getCategory() {
+        return Category.TROUBLESHOOTING;
+    }
+
+    @NonNull
+    @Override
+    public Permission getRequiredPermission() {
+        return Jenkins.MANAGE;
+    }
+
+    @RequirePOST
+    public void doDownloadDiagnosticFlare(StaplerRequest request, StaplerResponse response) throws Exception {
+        if (!Jenkins.get().hasPermission(Jenkins.MANAGE)) {
+            response.sendError(HttpServletResponse.SC_FORBIDDEN);
+            return;
+        }
+
+        try {
+            LocalDateTime now = LocalDateTime.now();
```

Comment:
Could we do this in UTC instead of Local time always?

Reply:
Makes sense, fixed